### PR TITLE
DOCS Add documentation and examples for all commands, and installation

### LINKED ISSUE #38: Add documentation examples for all commands
Raised in #37 

We need example documentation for all commands in the console.

### PR DESCRIPTION
**Pull request by robbieaverill**
Resolves #38 

cc @andrewandante - merry bananas!

### CODE REVIEW

File: docs/en/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+###  Configuration
+
+* [`config:dump`](commands/config-dump.md): Dumps all of the processed configuration properties and their values
+* [`config:get`](commands/config-get.md): Look up a specific configuration value
+
+### Development
+
+* [`dev:build`](commands/dev-build.md): Builds the SilverStripe database
+* [`dev:task:*`](commands/dev-tasks.md): Run a native SilverStripe task (automatically populated)
```
Comment (andrewandante) [inline docs/en/README.md]: Would prefer something like ['dev:task:{taskname}']

File: docs/en/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+### Membership
+
+* [`member:change-groups`](commands/member-change-groups.md): Change a member's groups
+* [`member:change-password`](commands/member-change-password.md): Change a member's password
+* [`member:create`](commands/member-create.md): Create a new member, and optionally add them to groups
+
+### Object debugging
+* [`object:children`](commands/object-children.md): List all child classes of a given class, e.g. "Page"
```
Comment (andrewandante) [inline docs/en/README.md]: Needs newline :)

File: docs/en/commands/object-debug.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
+# Command: `object:debug`
+
+Look up a DataObject by class name and either it's ID or an interactive search-by-column.
```
Comment (andrewandante) [inline docs/en/commands/object-debug.md]: Wrong its

File: docs/en/commands/object-debug.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+If no ID is provided then an interactive prompt will ask for the column to search by, then autocomplete all available
+values for that class's columns to choose from.
+
+The default output format is JSON. You can add the `--output-table` option to output the results in a table instead.
+
+By default the output will also be sorted by key. Do prevent this, add the `--no-sort` option.
```
Comment (andrewandante) [inline docs/en/commands/object-debug.md]: To prevent

**Comment by robbieaverill** [comment]
All updated, thanks @andrewandante

**Pull request merged**